src/Service/SF1500/FetchServiceInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Service\SF1500;
+
+interface FetchServiceInterface
```

Comment:
The interface names hints that it fetches a service, but it actually fetches data:

```suggestion
interface DataFetcherInterface
```

The implementing services should also be renamed, e.g. from `BrugerFetchService` to `BrugerDataFetcher`, say.